react-elements/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
For `ReactDOM` it should be the `client` version, and you don't need to use the `*`. 
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```

Reply:
Okay, thank you!